src/evolve_pressure.cxx

```diff
@@ -512,6 +538,35 @@ void EvolvePressure::outputVars(Options& state) {
                     {"long_name", name + " power through Y cell face. Note: May be incomplete."},
                     {"species", name},
                     {"source", "evolve_pressure"}});
+                    
+      set_with_attrs(state[std::string("ConductionFlow_") + name + std::string("_ylow")], flow_ylow_conduction,
+                   {{"time_dimension", "t"},
+                    {"units", "W"},
+                    {"conversion", rho_s0 * SQ(rho_s0) * Pnorm * Omega_ci},
+                    {"standard_name", "power"},
+                    {"long_name", name + " conduction through Y cell face. Note: May be incomplete."},
+                    {"species", name},
+                    {"source", "evolve_pressure"}});
+
+      set_with_attrs(state[std::string("KineticFlow_") + name + std::string("_ylow")], flow_ylow_kinetic,
+                   {{"time_dimension", "t"},
+                    {"units", "W"},
+                    {"conversion", rho_s0 * SQ(rho_s0) * Pnorm * Omega_ci},
+                    {"standard_name", "power"},
+                    {"long_name", name + " kinetic energy flow through Y cell face. Note: May be incomplete."},
+                    {"species", name},
+                    {"source", "evolve_pressure"}});
+    }
+
+    if (numerical_viscous_heating) {
+      set_with_attrs(state[std::string("SP") + name + std::string("_nvh")], Sp_nvh,
```

Comment:
To keep with the most common naming convention, this should be converted to energy and named E<species>_nvh.

Reply:
Unless we make the naming convention so that e.g. Ed+_nvh would be for terms relevant to balance or physics post-processing, where most variables are already in the form of energy, and e.g. `SPd+_nvh` would specifically refer to equation terms. The problem is that `S` already means density source.

Reply:
I just tested: `fmt::format("SP{}_nvh", name)` is around 40% faster then `std::string("SP") + name + std::string("_nvh")` :-)
<details>

```
[dave@raven04: ~]👍 cat /viper/u/dave/soft/hermes-2/relax/test.cc
#include <string>
#include <fmt/format.h>

int main() {
  const std::string name{"h+"};
  for (int i=0 ; i< 10000000; ++i) {
#ifdef FMT
    const auto fu = fmt::format("SP{}_nvh", name);
#else
    const auto fu = std::string("SP") + name + std::string("_nvh");
#endif
    asm("");
  }
}
[dave@raven04: ~]👍 g++ /viper/u/dave/soft/hermes-2/relax/test.cc -lfmt && time ./a.out

real    0m1.011s
user    0m1.010s
sys     0m0.000s
[dave@raven04: ~]👍 g++ -DFMT /viper/u/dave/soft/hermes-2/relax/test.cc -lfmt && time ./a.out

real    0m0.669s
user    0m0.665s
sys     0m0.004s
```
</details>